spec/fixtures/pages.yml

```diff
@@ -43,11 +47,11 @@ fashion_collection:
 
 grid_layout_collection:
   slug: 'collections/grid_layout'
+  collection: grid_layout
   state: 1
   type: 'Page::Landing'
 
 draft_landing_page:
```

Comment:
Is there any point in keeping this fixture as landing pages need to be associated with collections now?